tetrateci/test_1.7.sh

```diff
@@ -31,7 +31,7 @@ for package in $PACKAGES; do
   do
     echo "========================================================TRY $n========================================================"
     go test -count=1 -p 1 -test.v -tags=integ $package -timeout 30m --istio.test.select=-postsubmit,-flaky ${CLUSTERFLAGS} && break || echo "Test Failed: $package"
-    sudo rm -rf $(ls /tmp | grep istio)
+    sudo rm -rf -- $(ls /tmp | grep istio)
```

Comment:
To the echo statement above - can we add the package name as well?  Something like TRYING PACKAGE $PACKAGE - count $n